### PR TITLE
Assets 31139 - Landing page

### PR DESCRIPTION
**Pull request by mdickson-adbe**
Added three new blocks:

1. gmo-landing-page
2. gmo-teaser
3. gmo-footer

Also added a metadata property "hide-header" (and updated scripts.js header-loading logic) that allows us to implement a page without the adp header.

When assembled in a document they can replicate the landing page from the AEM Sites version of Marketing Hub/GenStudio.

Styling includes mobile breakpoints for adjusting font size/layout/the video that appears in the page background.

URL for testing:
- https://assets-31139--adobe-gmo--hlxsites.hlx.page/drafts/mdickson/landing-page

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -232,7 +232,9 @@ async function loadLazy(doc) {
   const element = hash ? doc.getElementById(hash.substring(1)) : false;
   if (hash && element) element.scrollIntoView();
 
-  loadHeader(doc.querySelector('header'), 'adp-header');
```
Comment (mdickson-adbe) [inline scripts/scripts.js:235]: @nithinasokan Would the scripts.js file be considered part of the immutable files? If so, can I raise a PR to include this change in the main ADP repo? It's a check for a metadata property "hide-header" that lets us create pages without the ADP header included

File: tools/sidekick/config.json [hunk 2]
```diff
@@ -1 +1,18 @@
-{}
+{
+  "project": "marketing-hub",
```
Comment (nithinasokan) [inline tools/sidekick/config.json:2]: @mdickson-adbe I'm unaware of what this change does; can you explain it for me? Thanks

File: scripts/scripts.js (hunk 1 above)
Comment (nithinasokan) [reply scripts/scripts.js:235]: Yes, scripts are immutable. Please submit a PR to adp repo and I'll approve it right away. Thanks.

**Review by nithinasokan — APPROVED** [review]
Please make sure to send `scripts` changes to ADP repo before merging this code.

File: scripts/scripts.js (hunk 1 above)
Comment (mdickson-adbe) [reply scripts/scripts.js:235]: https://github.com/adobe/assets-distribution-portal/pull/242 @nithinasokan

File: tools/sidekick/config.json (hunk 2 above)
Comment (mdickson-adbe) [reply tools/sidekick/config.json:2]: @nithinasokan this was included at @shiv-gup request, it's meant to provide a configuration to use the assets picker feature in the sidekick, so we can use assets from the gmo assets repo when building pages.

File: scripts/scripts.js (hunk 1 above)
Comment (nithinasokan) [reply scripts/scripts.js:235]: Merged and synced to GMO

**Pull request merged**